doc/All_Tags.dox

```diff
@@ -11,6 +12,64 @@ These filters are applied in various areas, such as:
 \li not showing buildings unless other criteria are met
 \li only affecting certain ships within a system.
 
+--
+
+@section content_tags_usage Alias usage
+
+Definition tags should be documented only once, at the first point of use within
+source files or scripts.
+Documentation within FOCS markup files is not yet
+supported.
+
+Some example code fragments:
+\verbatim // File: Sample1.h
+/** \content_tag{CTRL_SAMPLE1} Sample 1 **/
+int SomeFunction(std::string tag = "CTRL_SAMPLE1");
+\endverbatim
+
+\verbatim // File: Sample2.cpp
+std::string sample_tag = "CTRL_SAMPLE2"; /// \content_tag{CTRL_SAMPLE2} Sample 2
```

Comment:
Don't mandate awful styles in the examples.  Documentation comments belong above the item to document, not behind or below because of the reading direction and code width limitations.  Why width limitation? Because overlong lines make side-by-side comparisons hard to read.